### ISSUE TITLE
docs(README): update inlcudes

### ISSUE THREAD

**Issue by felipew** [assisrafael/angular-input-masks#84]
When adding via bower and using wiredep to auto-include the required files, the resulting in my index.html is the following
```
<script src="bower_components/br-validations/releases/br-validations.js"></script>
<script src="bower_components/string-mask/src/string-mask.js"></script>
<script src="bower_components/angular-input-masks/angular-input-masks.js"></script>
```

The readme mention a file not included in the lib:
```
How to use

Import the masks.min.js script in your page.
```

**Comment by assisrafael** [comment]
Yes, the README is incorrect.

**Comment by assisrafael** [comment]
Landed in v1.5.1.
